utils/fotos/update.py

```diff
@@ -41,24 +40,26 @@ def scan(album):
                     'name': name,
                     'random': random.random(),
                     'visibility': ['hidden']})
+                subalbum.update_metadata(album)
+                subalbum.save()
```

Comment:
Ik heb `save` wat consistenter gemaakt. Alle `update_*` functies saven nu 'by default', maar kunnen ook niet opslaan als dat nodig is (op `update_visibility` na omdat die complexer is, die slaat het altijd op).